src/main/java/frc/robot/RobotContainer.java

```diff
@@ -77,6 +99,26 @@ public RobotContainer() {
    * edu.wpi.first.wpilibj2.command.button.JoystickButton}.
    */
   private void configureButtonBindings() {
+    if (Robot.ledSubSelect == LedEnum.MATRIX){
+     OI.Operator.getLedToggleButton().onTrue(
+      new SequentialCommandGroup(
+        new InstantCommand(() -> ledMatrixSubsystem.serpentine(LedDriver.three), ledMatrixSubsystem),
+        new WaitCommand(1),
+        new InstantCommand(() -> ledMatrixSubsystem.serpentine(LedDriver.two), ledMatrixSubsystem),
+        new WaitCommand(1),
+        new InstantCommand(() -> ledMatrixSubsystem.serpentine(LedDriver.one), ledMatrixSubsystem),
+        new WaitCommand(1),
+        new InstantCommand(() -> ledMatrixSubsystem.serpentine(LedDriver.gocans), ledMatrixSubsystem))
+      );
```

Comment:
Create method in the subsystem that returns this Command. Then we still have it if we want to use it later. 
But the button mappings should just be toggling the colors on/off in each subsystem.

Reply:
resolved